be/src/olap/rowset/segment_v2/page_io.cpp

```diff
@@ -154,13 +154,11 @@ Status PageIO::read_and_decompress_page(const PageReadOptions& opts, PageHandle*
         opts.stats->compressed_bytes_read += page_size;
     }
 
-    if (opts.verify_checksum) {
```

Comment:
Why remove this if? Because it is always true?

Reply:
yes